examples_tests/48.ArithmeticUnitTest/testExclusive.comp

```diff
@@ -0,0 +1,15 @@
+#version 450
+#define _IRR_GLSL_WORKGROUP_SIZE_ ???? 
+
+#include "common.glsl"
+
+void main()
+{
+	andOutput [gl_LocalInvocationIndex] = irr_glsl_subgroupExclusiveAnd(inputValue[gl_LocalInvocationIndex]);
```

Comment:
`gl_LocalInvocationIndex` is the index within a workgroup, not within all dispatches